easybuild/easyconfigs/p/PETSc/PETSc-3.7.3-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,34 @@
+name = 'PETSc'
+version = '3.7.3'
+versionsuffix = '-Python-2.7.11'
+
+homepage = 'http://www.mcs.anl.gov/petsc'
+description = """PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the
+ scalable (parallel) solution of scientific applications modeled by partial differential equations."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['http://ftp.mcs.anl.gov/pub/petsc/release-snapshots']
+sources = [SOURCELOWER_TAR_GZ]
+
+patches = ['PETSc_ranlib-fix.patch']
+
+parmetis = 'ParMETIS'
+parmetis_ver = '4.0.3'
+dependencies = [
+    ('Boost', '1.61.0', versionsuffix),
+    ('ScientificPython', '2.9.4', versionsuffix),
+    ('FIAT', '1.6.0', versionsuffix),
+    ('METIS', '5.1.0'),
+    (parmetis, parmetis_ver),
+    ('SCOTCH', '6.0.4'),
+    ('SuiteSparse', '4.5.3', '-%s-%s' % (parmetis, parmetis_ver)),
+    ('Hypre', '2.11.1'),
+]
+
+builddependencies = [('CMake', '3.5.2')]
+
+configopts = '--LIBS="$LIBS" -lrt'
```

Comment:
@RvDijk sorry, this should be:

```python
configopts = '--LIBS="$LIBS -lrt"'
```

so that `-lrt` is actually included in the value passed to `--LIBS`...